Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2730,7 +2748,7 @@ extension BrowserViewController: TabManagerDelegate {
 }
 
 /// List of schemes that are allowed to be opened in new tabs.
-private let schemesAllowedToBeOpenedAsPopups = ["http", "https", "javascript", "about"]
+private let schemesAllowedToBeOpenedAsPopups = ["http", "https", "javascript", "about", "whatsapp"]
```

Comment:
nit: add a manual test for this new scheme.

Reply:
Tests are added for every url scheme